app/views/coronavirus_landing_page/components/shared/_announcements.html.erb

```diff
@@ -1,27 +1,25 @@
-<div class="govuk-grid-row">
-  <% announcements.each do |announcement| %>
-    <div class="govuk-grid-column-one-third covid__page-header-announcement">
-      <p class="govuk-body covid__inverse">
-        <% if announcement["href"].present? %>
-          <a
-            href="<%= announcement["href"] %>"
-            class="govuk-link covid__page-header-link"
-            data-module="track-click"
-            data-track-category="pageElementInteraction"
-            data-track-action="Announcements"
-            data-track-label="<%= announcement["href"] %>"
-          >
-            <%= announcement["text"] %>
-          </a>
-        <% else %>
+<ul class="govuk-list covid__announcements">
+  <% announcements.each do | announcement | %>
+    <li class="covid__topic-list-item">
+      <% if announcement["href"].present? %>
+        <a
+          href="<%= announcement["href"] %>"
+          class="govuk-link covid__topic-list-link"
+          data-module="track-click"
+          data-track-category="pageElementInteraction"
+          data-track-action="Announcements"
+          data-track-label="<%= announcement["href"] %>"
+        >
```

Comment:
Is it worth considering the [document list component](https://components.publishing.service.gov.uk/component-guide/document_list) for this section?

Reply:
I did think about it but it doesn't match the styling